File: w1/square1.c
#include <stdio.h>

int main() {
    int size;

    scanf("%d", &size);
    for ( int row = 1; row <= size; row++ ) {
        int col = 1;

        for ( ; col < size; col++ ) {
            printf("%d ", col);
        }
        printf("%d\n", col);
    }
    return 0;
}

File: w4/php/square1.php
<?php
    $size = intval (fgets(STDIN));
    
    for ( $row = 1; $row <= $size; $row++) {
        $out = '';
        for ( $col = 1; $col <= $size; $col++) {
            $out .= $col . ' ';
        }
        echo $out . PHP_EOL;
    }
    
    
?>
